tutorials/debugging/vscode-native/README.md

```diff
@@ -0,0 +1,268 @@
+# Develop and Debugging Holoscan Application Natively
+
+This tutorial will guide you through the steps to set up your development environment for building and debugging your applications using Holoscan SDK natively with VS Code.
+
+Whether you are a C++ or Python developer, you can use the tutorial as a reference to set up your development environment and debug your Holoscan application.
+
+> 💡 Note: This guide is specific to the Linux development environment and is tested on Ubuntu 22.04 LTS.
+
+## Prerequisites
+
+- [Holoscan SDK Prerequisites](https://docs.nvidia.com/holoscan/sdk-user-guide/sdk_installation.html#prerequisites)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Holohub](https://github.com/nvidia-holoscan/holohub/)
+  ```bash
+  # clone the Holohub repository
+  git clone https://github.com/nvidia-holoscan/holohub.git
+  ```
+
+### C++ Prerequisites
+
+To build and debug your C++ applications natively, we must first install the redistribution of the Holoscan SDK. To install Holoscan SDK using a Debian package, refer to the Debian package installation section in the [Holoscan SDK documentation](https://docs.nvidia.com/holoscan/sdk-user-guide/sdk_installation.html#instructions).
+
+- [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools) extension for VS Code
+- [CMake](https://cmake.org/cmake/help/latest/command/install.html) 3.20 or higher
```

Comment:
Is there a reason to use CMake 3.20? We distribute 3.24.0 in the Holoscan SDK dev container

Reply:
Let me change that to `3.24.0`